SearchBar/index.js

```diff
@@ -107,28 +120,36 @@ export default class SearchBar extends React.Component {
 
         const {
             options,
-            data
-        } = this.props
+            data,
+            autoSearch,
+        } = this.props;
 
         return (
             <div style={{display: "flex", margin: "5px"}}>
                 <div className="search-label">
                     Search: &nbsp;
                 </div>
                 <div className="filter search">
-                    <ReactFilterBox
-                        ref="filterBox"
-                        data={data}
-                        onChange={this.onChange}
-                        autoCompleteHandler={this.autoCompleteHandler}
-                        query={query}
-                        options={options}
-                        onParseOk={this.onParseOk}
-                    />
+                        <ReactFilterBox
+                            ref="filterBox"
+                            data={data}
+                            onChange={this.onChange}
+                            autoCompleteHandler={this.autoCompleteHandler}
+                            query={query}
+                            options={options}
+                            onParseOk={this.onParseOk}
+                        /> 
                     <div className="filter-icon">
                         { this.renderIcon() }
                     </div>
                 </div>
+               {autoSearch === false && <div style={{marginTop: "6px"}}> 
```

Comment:
if using FaSearch we need to increase the top margin to '10px' and icon size to 18